### PR TITLE
Activate population in low level

### PR DESCRIPTION
**Pull request by apragsdale**
Getting toward adding the de/activate population events to the Python side of things. I'm a bit hung up on an error that I haven't been able to figure out the source of last few days. In the new `test_activate_population()` in `test_lowlevel.py`, I get a `ValueError: 'population_id' not specified` when calling `sim = _msprime.Simulator()` within `make_sim()`. Feel like I'm missing something either very simple or very fundamental about what's happening in the low level function.

Edit.. weird - fails locally with that error, but passes CI here. so maybe nvm

### CODE REVIEW

**Comment by apragsdale** [comment]
I think this is in reasonable shape. Though I'm having a hard time hitting those last few lines with test coverage. Not exactly sure how to approach it.

File: msprime/_msprimemodule.c [hunk 1]
```diff
@@ -1242,6 +1242,30 @@ Simulator_parse_simulation_model(Simulator *self, PyObject *py_model)
...
+
+    err = msp_add_activate_population_event(self->sim, time, population_id);
+    if (err != 0) {
+        handle_input_error("activate population event", err);
+        goto out;
+    }
+    ret = 0;
+out:
+    return ret;
+
```
Comment (jeromekelleher) [inline msprime/_msprimemodule.c:1266]: Extra blank line here

**Review by jeromekelleher — COMMENTED** [review]
This is excellent @apragsdale, I just spotted one tiny formatting nit. Ready to merge after a squash.

Hitting all lines in the C code isn't feasible sometimes, I think you've done a great job of covering the possibilities here.

**Comment by apragsdale** [comment]
Ok, thanks @jeromekelleher!. Fixed up the small formatting and squashed.

**Review by jeromekelleher — APPROVED** [review]

**Pull request merged**